jumanji/environments/routing/pacman/env.py

```diff
@@ -0,0 +1,551 @@
+        taken, i.e. there is a wall blocking the action, then no action (no-op) is taken.
+
+    - reward: jax array (float32) of shape (): 10 per pellet collected, 20 for a power pellet
+        and 200 for each unique ghost eaten.
+
+    - episode termination (if any):
+        - agent has collected all pellets.
+        - agent killed by ghost.
+        - timer has elapsed.
+
+    - state: State:
+        - key: jax array (uint32) of shape(2,).
+        - grid: jax array (int)) of shape (31,28) of the ingame maze with walls.
+        - pellets: int tracking the number of pellets.
+        - frightened_state_time: jax array (int) of shape ()
+            tracks number of steps for the scatter state.
+        - pellet_locations: jax array (int) of pellets of shape (316,2).
+        - power_up_locations: jax array (int) of power-pellet locations of shape (4,2).
+        - player_locations: current 2D position of agent.
+        - ghost_locations: jax array (int) of ghost positions of shape (4,2).
+        - initial_player_locations: starting 2D position of agent.
+        - initial_ghost_positions: jax array (int) of ghost positions of shape (4,2).
+        - ghost_init_targets: jax array (int) of ghost positions.
+            used to direct ghosts on respawn.
+        - old_ghost_locations: jax array (int) of shape (4,2) of ghost positions from last step.
+            used to prevent ghost backtracking.
+        - ghost_init_steps: jax array (int) of shape (4,2) number of initial ghost steps.
+            used to determine per ghost initialisation.
+        - ghost_actions: jax array (int) of shape (4,).
+        - last_direction: int tracking the last direction of the player.
+        - dead: bool used to track player death.
+        - visited_index: jax array (int) of visited locations of shape (320,2).
+            used to prevent repeated pellet points.
+        - ghost_starts: jax array (int) of shape (4,2)
+            used to reset ghost positions if eaten
+        - scatter_targets: jax array (int) of shape (4,2)
+            target locations for ghosts when scatter behavior is active.
+        - step_count: (int32) of total steps taken from reset till current timestep.
+        - ghost_eaten: jax array (bool)of shape (4,) tracking if ghost has been eaten before.
+        - score: (int32) of total points aquired.
+
+
+
+    ```python
+    from jumanji.environments import pac_man
+    env = PacMan()
+    key = jax.random.key(0)
+    state, timestep = jax.jit(env.reset)(key)
+    env.render(state)
+    action = env.action_spec().generate_value()
+    state, timestep = jax.jit(env.step)(state, action)
+    env.render(state)
+    ```
+    """
+
+    def __init__(
+        self,
+        generator: Optional[Generator] = None,
+        viewer: Optional[Viewer[State]] = None,
+        time_limit: Optional[int] = None,
+    ) -> None:
+        """Instantiates a `Pac-Man` environment.
```

Comment:
Again, for consistency, we must use the same naming. In this case, we need to use the name of the class, i.e. `PacMan`.
```suggestion
        """Instantiates a `PacMan` environment.
```

Reply:
Updated